README.md

```diff
@@ -67,6 +67,53 @@ targets:
 Note that configuration files can be stored in any place, even in a local folder. We recommend to
 use a VCS (like git) to store them; treat them as source code.
 
+### Using Docker to build and run Copybara
+
+You can build copybara using Docker like so
+
+```
+docker build --rm -t copybara .
+```
+
+Once this has finished building you can run the image like so from the root of the code you are trying to use Copybara on:
+
+```
+docker run -it -v "$(pwd)":/usr/src/app copybara copybara
+
+```
+
+A few environment variables exist to allow you to change how you run copybara:
+* `COPYBARA_CONFIG=copy.bara.sky`
+  * allows you to specify a config name that isn't `copy.bara.sky`, defaults to `copy.bara.sky`
+* `COPYBARA_SUBCOMMAND=migrate`
```

Comment:
We might want to clarify that this is so that you can use other paths, but that the file name itself has to end with /copy.bara.sky (we have a check for that).

Reply:
Addressed in #45